pkgs/games/openttd/default.nix

```diff
@@ -32,11 +32,11 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "openttd";
-  version = "13.4";
+  version = "14.0";
 
-  src = fetchurl {
+  src = fetchzip {
```

Comment:
This was the only use of `fetchurl`, so if this becomes either `fetchzip` or `fetchFromGitHub` it should be removed from the input arguments.

Reply:
Yup, got rid of fetchurl

Reply:
I'd recommend fetchFromGitHub as it should be fetched from GitHub releases.
Anyways, please squash the commits into one.
By `git rebase --interactive HEAD~3`, set the newer two commits from `pick` to `squash`. Then force push the changes using `git push --force-with-lease`.

Reply:
> I'd recommend fetchFromGitHub as it should be fetched from GitHub releases.

Why is downloading from Github better than the CDN link advertised on the official webpage?

Reply:
Using `fetchzip` is wrong because a xz tarball is being downloaded. Could you please review PRs of first-time contributors with more care? This could've just been merged in its original version with a simple squash to rename the commit without your nitpicking.

Reply:
> Why is downloading from Github better than the CDN link advertised on the official webpage?

Fetching the source directly from GitHub feels more safe than fetching from a tarball provided by the distributor.

Fyi, I did not even ask the user to use `fetchzip` and of course there's a [difference](https://ryantm.github.io/nixpkgs/builders/fetchers/#fetchurl) between `fetchzip` and `fetchurl`.